packages/screens/AdministrationDashboard/AdministrationDashboardScreen.tsx

```diff
@@ -0,0 +1,333 @@
+import React, { useState } from "react";
```

Comment:
Maybe extract some components to don't have a file with 333 lines ?

Reply:
Fixed, PR updated